File: bm_ui_android_kotlin/app/src/main/java/de/colinsteffen/bm_ui_android_kotlin/View/Adapter/InformationAdapter.java
package de.colinsteffen.bm_ui_android_kotlin.View.Adapter;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;
import android.widget.BaseAdapter;
import android.widget.TextView;
import android.widget.TwoLineListItem;

import java.util.ArrayList;
import java.util.List;

import de.colinsteffen.bm_ui_android_kotlin.Model.Information;
import de.colinsteffen.bm_ui_android_kotlin.R;

public class InformationAdapter extends ArrayAdapter<Information> {

    public InformationAdapter(Context context, ArrayList<Information> informationList){
        super(context, 0, informationList);
    }

    @Override

    public View getView(int position, View convertView, ViewGroup parent) {
        Information information = getItem(position);

        if (convertView == null)
            convertView = LayoutInflater.from(getContext()).inflate(android.R.layout.simple_list_item_2, parent, false);

        TextView tvStart = (TextView) convertView.findViewById(android.R.id.text1);
        TextView tvEnd = (TextView) convertView.findViewById(android.R.id.text2);

        tvStart.setText(information.getStart());
        tvEnd.setText(information.getEnd());

        return convertView;
    }
}

File: bm_ui_xam_platform/bm_ui_xam_platform/View/InformationAdapter.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Java.Lang;
using Shared.Model;
using static Android.Support.V7.Widget.RecyclerView;

namespace bm_ui_xam_platform.View
{
    public class InformationAdapter : BaseAdapter
    {
        List<Information> InformationList { get; set; }

        public InformationAdapter(List<Information> informationList)
        {
            this.InformationList = informationList;
        }

        public override int Count => InformationList.Count;

        public override Java.Lang.Object GetItem(int position)
        {
            return null;
        }

        public override long GetItemId(int position) => 0;

        public override Android.Views.View GetView(int position, Android.Views.View convertView, ViewGroup parent)
        {
            var view = convertView;

            if(view == null)
            {
                view = LayoutInflater.From(parent.Context).Inflate(Resource.Layout.listview_item, parent, false);

                var start = view.FindViewById<TextView>(Resource.Id.starttext);
                var end = view.FindViewById<TextView>(Resource.Id.endTextView);

                start.Text = InformationList[position].Start;
                end.Text = InformationList[position].End;
            }

            return view;
        }
    }
}